lua/bbye.lua

```diff
@@ -0,0 +1,180 @@
+-- Bbye (rewritten in Lua)
+--
+-- source: https://github.com/moll/vim-bbye/blob/master/plugin/bbye.vim
+-- license:
+--
+-- Copyright (C) 2013 Andri Möll
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or any later version.
+--
+-- Additional permission under the GNU Affero GPL version 3 section 7:
+-- If you modify this Program, or any covered work, by linking or
+-- combining it with other code, such other code is not for that reason
+-- alone subject to any of the requirements of the GNU Affero GPL version 3.
+--
+-- In summary:
+-- - You can use this program for no cost.
+-- - You can use this program for both personal and commercial reasons.
+-- - You do not have to share your own program's code which uses this program.
+-- - You have to share modifications (e.g bug-fixes) you've made to this program.
+--
+-- For the full copy of the GNU Affero General Public License see:
+-- http://www.gnu.org/licenses.
+
+-------------------
+-- Section: helpers
+-------------------
+
+--- Use `vim.notify` to print an error `msg`
+--- @param msg string
+local function err(msg)
+  vim.notify(msg, vim.log.levels.ERROR, {title = 'bbye'})
```

Comment:
`title` is not an official part of `vim.notify`, but that will give it integration with the `nvim-notify` plugin.